### DISCUSSION TITLE
美髮業運用AI精準行銷，吸引潛在顧客上門

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#692] [General]
在全台擁有多分店的名留國際與AI新創團隊媒合，強化廣告觸及率，提升廣告效益。
首先，先搜集顧個資料，再進一步整合名留以往在臉書投放廣告的歷史紀，來預測廣告投放的觸及率
除此之外，美髮業也有淡旺季顧客的變化，把這個很特別的資料交給AI分析，可找出既有盲點，改善行銷效益。
這個美髮店與AI團隊的結合真的很有趣，因為很少聽到美髮業也會用到AI的技術。我覺得比較有趣的是我前陣子在分析一比要預測銷售額的資料時，也有發現超市也有特定商品的淡旺季之分，需特別拉出來比較，沒想到美髮業也有這種淡旺季的差別，太酷了！

文章來源：https://www.thenewslens.com/feature/aishow/177165

**Comment by 0212Jerry** [comment]
Ai 在預測與判斷方面的水準逐漸提升，人類彷彿找到了一片新的良田，正以一定的速度開闢，相信未來會有更多Ai 應用上的傑作出現，不僅美髮業，其他的各行各業也有類似的趨勢。此外，新的產業也逐漸起步。

**Comment by Vincent20030314** [comment]
AI的建議確實不錯，這樣可以增加店家的經營效率。

**Comment by HsuChen88** [comment]
說個題外話，如果有個手機APP可以分析髮型的比例，這樣去剪百元快剪也不用怕阿姨亂剪，只要照著手機上的比例剪就行了。

**Comment by yjpxi** [comment]
我覺得超市商品淡旺季分析和美髮業的淡旺季顧客變化，這兩者之間的相似性正好呈現了 AI 在不同領域中的靈活應用。AI 可以通過歷史數據分析來預測未來趨勢，這對於那些希望精確打擊目標市場的商業模式來說，是很大的優勢。
